pytket-examples-book/_toc.yml

```diff
@@ -0,0 +1,30 @@
+# Table of contents
+# Learn more at https://jupyterbook.org/customize/toc.html
+
+format: jb-book
+root: README
+chapters:
```

Comment:
Do you mean in this file? Or should we have categories that the user can click through in the built pages?

Reply:
<img width="307" alt="image" src="https://github.com/CQCL/pytket/assets/75305462/b818fcaf-17fd-48ea-ab95-d041cf667a44">
I mean the table of content here. Not the easiest to navigate, but I suppose we can improve it later.

Reply:
I see thanks.

I'm planning on making some revisions to the examples soon. Do you think it would make sense to categorise the examples in a future PR? I think some may be outdated and should probably be removed/changed.